tools/bowtie2/bowtie2_wrapper.xml

```diff
@@ -145,18 +145,19 @@
         
         #elif str( $analysis_type.analysis_type_selector ) == "cline":
             ${analysis_type.cline}
-        #end if    
+        #end if
         
+        -S "galaxy_bowtie_2_output.sam" &amp;&amp;
+
         ## view/sort and output BAM file
-        | samtools view -Su - | samtools sort -o - - > $output
+        ( samtools view -Su "galaxy_bowtie_2_output.sam" | samtools sort -o - - > $output )
 
         ## rename unaligned sequence files
         #if $library.type == "paired" and $output_unaligned_reads_l and $output_unaligned_reads_r:
-            #set left  = str($output_unaligned_reads_l).replace( '.dat', '.1.dat' )
-            #set right = str($output_unaligned_reads_l).replace( '.dat', '.2.dat' )
-        
-            ; mv $left $output_unaligned_reads_l;
-            mv $right $output_unaligned_reads_r
+            #set left  = str($output_unaligned_reads_l)[:-3] + '1.dat'
+            #set right = str($output_unaligned_reads_l)[:-3] + '2.dat'
```

Comment:
I haven't tested it but the `1.dat` looks strange to me. Is bowtie2 creating `*.dat` files?